<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintTemplateEditorDialog</class>
 <widget class="QDialog" name="PrintTemplateEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="10" column="0" colspan="5">
    <widget class="QFrame" name="frmExaro">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>3</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="lblFileName">
        <property name="text">
         <string>Имя &amp;файла</string>
        </property>
        <property name="buddy">
         <cstring>edtFileName</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btnEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Изменить шаблон Exaro во внешнем редакторе</string>
        </property>
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="edtFileName">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="5">
    <widget class="QGroupBox" name="grpDefault">
     <property name="title">
      <string>Шаблон</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTextEdit" name="edtDefault"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtContext"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>На&amp;именование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="edtName">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblType">
     <property name="text">
      <string>Тип</string>
     </property>
     <property name="buddy">
      <cstring>cmbType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblContext">
     <property name="text">
      <string>Кон&amp;текст</string>
     </property>
     <property name="buddy">
      <cstring>edtContext</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cmbDPD">
     <item>
      <property name="text">
       <string>Не меняет</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Меняет на &quot;Да&quot;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Меняет на &quot;Нет&quot;</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="cmbType">
     <item>
      <property name="text">
       <string>HTML</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exaro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SVG</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblDPD">
     <property name="text">
      <string>Меняет ДПД</string>
     </property>
     <property name="buddy">
      <cstring>cmbDPD</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QComboBox" name="cmbBanUnkeptData">
     <property name="toolTip">
      <string>На данный момент распространяется только на обращения. Доступно 3 варианта: &lt;ul&gt;&lt;li&gt;&lt;i&gt;Разрешено&lt;/i&gt; - печать доступна в любой момент.&lt;/li&gt;&lt;li&gt;&lt;i&gt;Запрещена&lt;/i&gt; - пока данные не будут сохранены, шаблоны не будут доступны для выбора.&lt;/li&gt;&lt;li&gt;&lt;i&gt;Запрещена, упрощенный режим сохранения&lt;/i&gt; - до сохранения данных шаблоны доступны для выбора, однако при их выборе будет выведено предупреждение с предложением сохранить данные. Пока данные не будут успешно сохранены, шаблон напечатан не будет.&lt;/li&gt;&lt;/ul&gt;</string>
     </property>
     <item>
      <property name="text">
       <string>Разрешена</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Запрещена</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Запрещена, доступен упрощенный режим сохранения</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>133</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="4">
    <widget class="CRBComboBox" name="cmbCounter">
     <property name="toolTip">
      <string>На данный момент распространяется только на обращения. Доступно 3 варианта: &lt;ul&gt;&lt;li&gt;&lt;i&gt;Разрешено&lt;/i&gt; - печать доступна в любой момент.&lt;/li&gt;&lt;li&gt;&lt;i&gt;Запрещена&lt;/i&gt; - пока данные не будут сохранены, шаблоны не будут доступны для выбора.&lt;/li&gt;&lt;li&gt;&lt;i&gt;Запрещена, упрощенный режим сохранения&lt;/i&gt; - до сохранения данных шаблоны доступны для выбора, однако при их выборе будет выведено предупреждение с предложением сохранить данные. Пока данные не будут успешно сохранены, шаблон напечатан не будет.&lt;/li&gt;&lt;/ul&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>131</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="edtGroup"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblGroup">
     <property name="text">
      <string>Группа</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="CRBComboBox" name="cmbIEMKDoc"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Тип документа</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="chkPatientApply">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>30</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>236</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Нужно подтверждение пациента</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lblBanUnkeptData">
     <property name="toolTip">
      <string>На данный момент распространяется только на обращения. Доступно 3 варианта: &lt;ul&gt;&lt;li&gt;&lt;i&gt;Разрешено&lt;/i&gt; - печать доступна в любой момент.&lt;/li&gt;&lt;li&gt;&lt;i&gt;Запрещена&lt;/i&gt; - пока данные не будут сохранены, шаблоны не будут доступны для выбора.&lt;/li&gt;&lt;li&gt;&lt;i&gt;Запрещена, упрощенный режим сохранения&lt;/i&gt; - до сохранения данных шаблоны доступны для выбора, однако при их выборе будет выведено предупреждение с предложением сохранить данные. Пока данные не будут успешно сохранены, шаблон напечатан не будет.&lt;/li&gt;&lt;/ul&gt;</string>
     </property>
     <property name="text">
      <string>Печать несохраненных данных</string>
     </property>
     <property name="buddy">
      <cstring>cmbBanUnkeptData</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="lblCounter">
     <property name="toolTip">
      <string>На данный момент распространяется только на обращения. Доступно 3 варианта: &lt;ul&gt;&lt;li&gt;&lt;i&gt;Разрешено&lt;/i&gt; - печать доступна в любой момент.&lt;/li&gt;&lt;li&gt;&lt;i&gt;Запрещена&lt;/i&gt; - пока данные не будут сохранены, шаблоны не будут доступны для выбора.&lt;/li&gt;&lt;li&gt;&lt;i&gt;Запрещена, упрощенный режим сохранения&lt;/i&gt; - до сохранения данных шаблоны доступны для выбора, однако при их выборе будет выведено предупреждение с предложением сохранить данные. Пока данные не будут успешно сохранены, шаблон напечатан не будет.&lt;/li&gt;&lt;/ul&gt;</string>
     </property>
     <property name="text">
      <string>Счётчик</string>
     </property>
     <property name="buddy">
      <cstring>cmbCounter</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtContext</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>cmbDPD</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>cmbBanUnkeptData</tabstop>
  <tabstop>cmbCounter</tabstop>
  <tabstop>edtFileName</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>edtDefault</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrintTemplateEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrintTemplateEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
